src/components/buttons/HighlightButton.tsx

```diff
@@ -42,7 +42,7 @@ const HighlightButton = ({ title, wide, ...props }: HighlightButtonProps) => {
       <Button
         title={title}
         style={{ backgroundColor: theme.global.accent }}
-        color={theme.font.primary}
+        color="white"
```

Comment:
I want to make sure to have max contrast for the highlight button, hence the constant.